content/docs/reference/config/helm-customization/index.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Customise the Helm Installation
+description: Customise the Istio installation with the included Helm chart.
+weight: 30
+keywords: [kubernetes,helm]
+aliases:
+    - /docs/reference/config/helm/index.html
+---
+
+The Helm chart ships with reasonable defaults.  There may be circumstances in which defaults require overrides.
```

Comment:
You can install Istio using Helm (provide link to install page), and although the included Helm chart uses reasonable defaults, there are circumstances that require customization. You can use Helm's `--set key=value` option to override one or more values. The set of supported keys is shown in the table below.  

The Istio helm chart includes reasonable default.

Reply:
"The Istio helm chart includes reasonable default." is already mentioned in the first sentence so I suggest have only your first paragraph.